terraform/aws/development-infrastructure/nbs-legacy/Docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# This Dockerfile is used to build NBS 6.0 Image locally and pushed to a ECR Repository
+
+# Use Windows Server Core as the base image
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+
+ENV ZIP_NAME="wildfly-10.0.0.Final-6.0.15.1"
```

Comment:
Moving this value to a default environment variable works. We can overwrite this as needed upon building the container. This type of setup only works for local builds that have access to that .zip.